index.md

```diff
@@ -34,11 +38,21 @@ Every large software development project relies on it,
 and most programmers use it for their small jobs as well.
 And it isn't just for software:
 books (like this one),
-papers,
+papers, theses, blogs,
 small data sets,
 and anything that changes over time or needs to be shared
 can and should be stored in a version control system.
 
+> ## Warning {.callout}
+> 
+> Git works best with plain text files that have regular line breaks.
+> This is generally true for programming code and marked up text.
+> 
+> It's considered a bad idea to try to use git with other filetypes,
```

Comment:
We use Git, with capital g, when refer to the "protocol" and `git` when refer to the program/implementation. So

~~~
It's considered a bad idea to try to use Git with other filetypes,
~~~